x-pack/plugins/cases/public/components/all_cases/all_cases_list.tsx

```diff
@@ -81,7 +81,7 @@ export const AllCasesList = React.memo<AllCasesListProps>(
     const hasOwner = !!owner.length;
     const firstAvailableStatus = head(difference(caseStatuses, hiddenStatuses));
```

Comment:
I know it is not your changes but maybe you looked into it 😅  

Do you know what this `hiddenStatus` logic is?

Reply:
I assumed stuff by I don't know exactly, sorry 🤷